### PR TITLE
[4.0] administrator/language/com_cpanel

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request für Issue #804 .

### Zusammenfassung der Änderungen

string ordering
translation

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

backend

### CODE REVIEW

File: administrator/language/de-DE/com_cpanel.ini [hunk 1]
```diff
@@ -5,38 +5,37 @@
...
 COM_CPANEL_MSG_UPDATEDEFAULTSETTINGS_TITLE="Aktualisierte Sicherheitsempfehlungen"
+COM_CPANEL_TITLE_SYSTEM_PANEL="System Panel"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_cpanel.ini:36]: @joomlagerman/reviewer we need a translation

File: administrator/language/de-DE/com_cpanel.ini [hunk 2]
```diff
@@ -5,38 +5,37 @@
...
-COM_CPANEL_HEADER_SUBMENU="Untermenü"
-COM_CPANEL_HEADER_SYSTEM="System"
-COM_CPANEL_LINK_CHECKIN="Globales Freigeben"
-COM_CPANEL_LINK_CLEAR_CACHE="Cache leeren"
-COM_CPANEL_LINK_DASHBOARD="Kontrollzentrum"
-COM_CPANEL_LINK_EXTENSIONS="Erweiterungen installieren"
-COM_CPANEL_LINK_GLOBAL_CONFIG="Konfiguration"
-COM_CPANEL_LINK_SYSINFO="Systeminformationen"
-COM_CPANEL_MESSAGES_BODY_NOCLOSE="Es gibt wichtige Nachinstallationshinweise die einer Aufmerksamkeit bedürfen. Zum Anzeigen der Hinweise bitte auf „Hinweise anzeigen“ klicken."
+COM_CPANEL_ADD_DASHBOARD_MODULE="Module zum Kontrollzentrum hinzufügen"
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_cpanel.ini:8]: from here to line 18 i'm not sure it represents the correct meaning. lets see how it looks in beta.

File: administrator/language/de-DE/com_cpanel.ini (hunk 1 above)
Comment (zero-24) [reply administrator/language/de-DE/com_cpanel.ini:36]:
```suggestion
COM_CPANEL_TITLE_SYSTEM_PANEL="Kontrollzentrum"
```

??

**Comment by zero-24** [comment]
Conflicting files @tecpromotion

**Comment by tecpromotion** [comment]
> Conflicting files @tecpromotion

@zero-24 please resolve the conflict

**Pull request merged**